calico/acl_manager/test/test_processor.py

```diff
@@ -53,6 +54,22 @@ def tearDown(self):
                          'outbound': [],
                          'outbound_default': 'deny'}}
 
+    def assertDictEqual(self, d1, d2, msg=None):
+        """
+        This override for assertDictEqual ensures that lists do not need to
+        be ordered for dict comparison. It's an unfortunate hack, but c'est la
+        vie.
+        """
+        for k,v1 in d1.iteritems():
```

Comment:
This only tests if d1 is a subdictionary of d2.  Please add the reverse comparison (nicest to just loop this?).